test/partiql-tests-runner/src/test/kotlin/org/partiql/runner/util/DatumUtil.kt

```diff
@@ -0,0 +1,63 @@
+package org.partiql.runner.util
+
+import com.amazon.ion.Decimal
+import com.amazon.ionelement.api.IonElement
+import com.amazon.ionelement.api.StructField
+import com.amazon.ionelement.api.field
+import com.amazon.ionelement.api.ionBool
+import com.amazon.ionelement.api.ionDecimal
+import com.amazon.ionelement.api.ionFloat
+import com.amazon.ionelement.api.ionInt
+import com.amazon.ionelement.api.ionListOf
+import com.amazon.ionelement.api.ionNull
+import com.amazon.ionelement.api.ionString
+import com.amazon.ionelement.api.ionStructOf
+import org.partiql.spi.types.PType
+import org.partiql.spi.value.Datum
+
+// For error display only
+internal fun Datum.toIonElement(): IonElement {
+    return when {
+        this.isNull() -> ionNull()
+        this.isMissing() -> ionNull()
```

Comment:
Think this should be the null annotated with the missing annotation `$missing::null`